src/app/lib/dropdown/dropdown.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'hc-dropdown',
+  templateUrl: './dropdown.component.html',
+  styleUrls: ['./dropdown.component.scss']
+})
+export class DropdownComponent implements OnInit {
```

Comment:
This is an angular deep track, but I believe we'll want to integrate this dropdown with reactive forms by implementing the `ControlValueAccessor` interface. 
[Custom Form Controls](https://alligator.io/angular/custom-form-control/)